test/specs/WebexMeeting/WebexMeetingPage.js

```diff
@@ -0,0 +1,95 @@
+import {createTestUser, removeTestUser} from '@webex/test-users';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+class WebexMeetingPage {
+  constructor() {
+    this.user = null;
+  }
+
+  setUser(user) {
+    this.user = user;
+  }
+
+  get accessTokenInput() {
+    return $('#token');
+  }
+  get destinationInput() {
+    return $('#destination');
+  }
+  get saveTokenBtn() {
+    return $('#md-button-1');
+  }
+  get displayWidgetBtn() {
+    return $('#md-button-3');
+  }
+  get microphoneIcon() {
+    return $('.meeting-controls-container').$$('button')[0];
+  }
+  get videoIcon() {
+    return $('.meeting-controls-container').$$('button')[1];
+  }
+  get videoRef() {
+    return $('video');
+  }
+  get joinMeetingBtn() {
+    return $('button=Join meeting');
+  }
+  get waitingForOthers() {
+    return $('h4=Waiting for others to join...');
+  }
+  get meetingContainer() {
+    return $('#wxc-interstitial-meeting');
+  }
+
+  open() {
+    browser.url('https://webex.github.io/widgets/');
```

Comment:
please put this in an env var `WEBEX_TEST_PAGE_URL` and move it to the top under the constructor